compliance/google/long_stopped_instances/google_long_stopped_instances.pt

```diff
@@ -64,214 +118,495 @@ pagination "google_pagination" do
   end
 end
 
-#get all google project
-datasource "ds_google_project" do
+###############################################################################
+# Datasources & Scripts
+###############################################################################
+
+# Table to derive region from zone
+datasource "ds_zone_to_region" do
+  run_script $js_zone_to_region
+end
+
+script "js_zone_to_region", type:"javascript" do
+  result "result"
+  code <<-EOS
+  result = {
+    "us-east1-b": "us-east1",
```

Comment:
No `us-east1-a` ?

Can see it isn't here either, just seems strange - https://cloud.google.com/compute/docs/regions-zones

Reply:
Nvm lol - https://www.reddit.com/r/googlecloud/comments/14169sp/gcp_reporting_to_use_useast1a_as_a_workaround_but/